docs/guides/agoda-affiliat-api-v2.0.md

```
@@ -0,0 +1,285 @@
+
+# HTTPS
+
+To utilize https, please contact your account manager for more information.
+
+---
+
+# URL
+
+Main endpoint:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)